docs/dictionary/property/icon.lcdoc

```diff
@@ -46,23 +46,21 @@ the <OS X> dock.
 
 LiveCode looks for the specified image in the following order:
 
-1) The stack of the object's <behavior> (if applicable)
-2) The stack of the owner of the object's <behavior> (if applicable)
-
-    ...
-
-n) The stack of the object's stack's <behavior> (if applicable)
-A) The object's stack
-B) The object's stack's mainstack (if a substack)
-C) The object's stack's mainstacks substacks
-D) The list of open stacks, in order they were loaded
+* 1. The stack of the object's <behavior> (if applicable)
+* 2. The stack of the owner of the object's <behavior> (if applicable)
+*    ...
+* n. The stack of the object's stack's <behavior> (if applicable)
+* a. The object's stack
+* b. The object's stack's mainstack (if a substack)
+* c. The object's stack's mainstack's substacks
+* d. The list of open stacks, in order they were loaded
```

Comment:
I think capital A,B,C,D works better here to distinguish the from the `n` of the previous part